### PR TITLE
fix: Movement while hacking

### PR DESCRIPTION
**Pull request by OriolSors**

### CODE REVIEW

File: Game/Assets/Scenes/_LEVEL02_ALFA.axolotl [hunk 1]
```diff
@@ -28824,15 +28824,15 @@
...
-                                }, 
-                                { 
-                                    "name": "DashDuration", 
-                                    "value": 0.30000000298023224, 
-                                    "type": 0 
+                                    "type": 0
+                                },
+                                {
+                                    "name": "DashRollDuration",
+                                    "value": 0.20000000298023224,
```
Comment (MarcGonzalezMoratona) [inline Game/Assets/Scenes/_LEVEL02_ALFA.axolotl:28834]:
```suggestion
                                    "value": 0.30000000298023224, 
```

this is a mistake on the name of a script that I did modifying the JSON in another PR, please fix it!

File: Game/Assets/Scenes/_LEVEL02_ALFA.axolotl [hunk 2]
```diff
@@ -106930,15 +106925,15 @@
...
```
Comment (MarcGonzalezMoratona) [inline Game/Assets/Scenes/_LEVEL02_ALFA.axolotl:106935]: same here!
```suggestion
                                    "value": 0.30000000298023224, 
```

**Comment by MarcGonzalezMoratona** [comment]
The only issue I found is that if we spam the attack buttons inside the hacking area and enable the hacking during the attack animation, it keeps shooting during the hacking

File: Game/Scripts/PlayerHackingUseScript.cpp [hunk 3]
```diff
@@ -197,28 +205,13 @@ void PlayerHackingUseScript::RestartHack()
...
-	PlayerManagerScript* manager = GetOwner()->GetComponentInternal<PlayerManagerScript>();
-	PlayerJumpScript* jump = GetOwner()->GetComponentInternal<PlayerJumpScript>();
-	PlayerMoveScript* move = GetOwner()->GetComponentInternal<PlayerMoveScript>();
-	PlayerAttackScript* attack = GetOwner()->GetComponentInternal<PlayerAttackScript>();
-	manager->Enable();
-	jump->Enable();
-	move->Enable();
-	attack->Enable();
+	playerManager->ParalyzePlayer(false);
+
```
Comment (MarcGonzalezMoratona) [inline Game/Scripts/PlayerHackingUseScript.cpp:214]:
```suggestion
```

**Review by MarcGonzalezMoratona — APPROVED** [review]
Works well with my controller. No issues found combining the hacking with other actions. Both success and failure hacking states are working.

**Review by markmeliaa — APPROVED** [review]
Thanks for the quick fix Oriol, now it works perfect!

**Pull request merged**